upgrades/tests/test_cluster.py

```diff
@@ -0,0 +1,163 @@
+import pytest
+import unittest.mock as mock
+
+import upgrade_testing_framework.cluster_management.cluster as cluster
+
+def test_WHEN_create_Cluster_AND_not_enough_nodes_THEN_raises():
+    # Set up test
+    test_cluster_name = "cluster-name"
+    test_num_nodes = 0 # must be > 1
```

Comment:
Minor point: `# must be >= 1`

at least according to the logic here: https://github.com/opensearch-project/opensearch-migrations/pull/40/files#diff-4409f008baff38f77bc0c7eb477861e6df14bf9abcac1a01355a68922ce0c68eR37

Reply:
Ah, good callout

Reply:
Addressed in next rev